catkit/hub/readme.org

```diff
@@ -0,0 +1,37 @@
+#+Title:CatHub: Catalysis-Hub.org interface
```

Comment:
Feel free to use whichever file format you prefer for documentation.

At some point, I will be moving all documentation which is not the package readme to be organized into the docs/ file. However, I haven't gotten this to work yet.